R/toolAggregate.R

```diff
@@ -120,7 +120,15 @@ toolAggregate <- function(x, rel, weight=NULL, from=NULL, to=NULL, dim=1, wdim=N
         if (length(from) == 0) {
           maxMatchColumn <- which.max(sapply(lapply(rel,intersect,items),length))
           unmappedItems <- setdiff(items, rel[[maxMatchColumn]])
-          stop(paste("The following items were not found in the mapping:", toString(unmappedItems)))
+          missingItems <- setdiff(rel[[maxMatchColumn]], items)
+
+          if (length(unmappedItems) > 0) {
+            warning(paste("The following items were not found in the mapping file:", toString(unmappedItems)))
```

Comment:
I would not call it "mapping file", just "mapping"

Reply:
In Zeile 107 steht "Cannot find given region mapping file!", ich änder das mal überall.

Reply:
Hm nein, dort geht es tatsächlich um ein file. Also mach ich die Änderung doch nur an dieser einen Stelle.